code/game/objects/items/blueprints.dm

```diff
@@ -25,12 +26,16 @@
 
 /obj/item/areaeditor/Topic(href, href_list)
 	if(..())
-		return
-	if(!usr.canUseTopic(src))
+		return TRUE
+	if(!usr.canUseTopic(src) || QDELETED(src))
```

Comment:
I feel like a QDELETED check should be higher up the call chain. You sure this isn't in ..() or /client/Topic()?

Reply:
it doesnt

Reply:
#36666